src/coreComponents/finiteElement/elementFormulations/Qk_Hexahedron_Lagrange_GaussLobatto.hpp

```diff
@@ -377,6 +403,39 @@ class Qk_Hexahedron_Lagrange_GaussLobatto final : public FiniteElementBase
                         real64 const (&X)[numNodes][3],
                         FUNC && func );
 
+  /**
+   * @brief computes the non-zero contributions of the d.o.f. indexd by q to the
+   *   stiffness matrix R for the elastic case, i.e., the superposition matrix of first derivatives
+   *   of the shape functions.
+   * @param q The quadrature point index
+   * @param X Array containing the coordinates of the support points.
+   * @param func Callback function accepting three parameters: i, j and R_ij
+   */
+  template< typename FUNC >
+  GEOSX_HOST_DEVICE
+  static void
+  computeFirstOrderStiffnessTerm( int q,
+                                  real64 const (&X)[numNodes][3],
+                                  FUNC && func );
```

Comment:
The callback functions are always a little bit cryptic. Maybe you could a comment to explain what it can be used for, and you could try a more meaningful name than `func` (that is not always easy)

Reply:
I'll try to update the comment. As you said it is often not easy to describe a callback...